### PR TITLE
store result of team getDetails in teamDetails obj as well

### PR DESCRIPTION
**Pull request by buoyad**
cc @keybase/y2ksquad

### CODE REVIEW

File: shared/teams/team/invites-tab/invite-row/container.tsx [hunk 1]
```diff
@@ -12,7 +12,6 @@ type OwnProps = {
 const mapStateToProps = (state, {teamname}: OwnProps) => {
   return {
     _invites: getTeamInvites(state, teamname),
-    _members: getTeamMembers(state, teamname),
```
Comment (buoyad) [inline shared/teams/team/invites-tab/invite-row/container.tsx]: unused

**Comment by buoyad** [comment]
@keybase/react-hackers bump

File: shared/actions/json/teams.json [hunk 2]
```diff
@@ -193,12 +193,13 @@
...
-      "members": "I.Map<string, Types.MemberInfo>",
-      "settings": "Types.TeamSettings",
-      "invites": "I.Set<Types.InviteInfo>",
-      "subteams": "I.Set<Types.Teamname>",
-      "requests": "I.Map<string, I.Set<Types.RequestInfo>>"
+      "members": "RPCTypes.TeamMembersDetails",
+      "settings": "RPCTypes.TeamSettings",
+      "invites": "Array<Types._InviteInfo>",
+      "subteams": "Array<Types.Teamname>",
+      "requests": "{[key: string]: Array<string>}"
```
Comment (chrisnojima) [inline shared/actions/json/teams.json]: could just be a `Map`

File: shared/constants/teams.tsx [hunk 3]
```diff
@@ -594,18 +590,28 @@ export const isOnTeamsTab = () => {
...
+  td ? Object.assign({...emptyTeamDetails}, td) : emptyTeamDetails
+
 export const teamListToDetails = (
   list: Array<RPCTypes.AnnotatedMemberInfo>
 ): Map<Types.TeamID, Types.TeamDetails> => {
   return new Map(
     list.map(t => [
       t.teamID,
-      {
+      makeTeamDetails({
```
Comment (chrisnojima) [inline shared/constants/teams.tsx]: this overrides all the props so it doesn't even need to call makeTeamDetails

**Comment by buoyad** [comment]
Thanks @chrisnojima, addressed feedback

**Pull request merged**